src/main/resources/biome-maps/thorium_v1.14.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "value": 1,
```

Comment:
I saw your comment on how this was supposed to be a minmax value, BiomeMaps actually support that too.
You can store any kind of data there, you will just need to write a short method to convert the `JsonElement` into an object.

```javascript
{
  "value": {
    "min": 1,
    "max": 3
  }
}
```

Reply:
Doesnt the deviation do this?